docs/management/connectors/action-types/servicenow-sir.asciidoc

```diff
@@ -1,12 +1,70 @@
 [role="xpack"]
 [[servicenow-sir-action-type]]
-=== ServiceNow connector and action
+=== ServiceNow SecOps connector and action
 ++++
 <titleabbrev>ServiceNow SecOps</titleabbrev>
 ++++
 
 The ServiceNow SecOps connector uses the https://docs.servicenow.com/bundle/orlando-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html[V2 Table API] to create ServiceNow security incidents.
 
+[float]
+[[servicenow-sir-connector-prerequisites]]
+==== Prerequisites
+After upgrading to 7.16, you must complete the following within your ServiceNow instance before creating a new ServiceNow SecOps connector or <<servicenow-sir-connector-update, updating an existing one>>:
+
+* Install [Elastic for Security Operations (SecOps)] https://store.servicenow.com/sn_appstore_store.do#!/store/application/2f0746801baeb01019ae54e4604bcb0f from the ServiceNow Store.
+* Create a ServiceNow integration user and assign it the appropriate roles.
+* Create a CORS rule.
+
+*Create a ServiceNow integration user*
+
+To ensure authenticated communication between Elastic and ServiceNow, create a ServiceNow integration user and assign it the appropriate roles. 
+
+. Go to *System Security -> Users*.
+. Click *New*.
+. Complete the form.
+. Assign the integration user the following roles: 
+* `import_set_loader`
+* `import_transformer`
+* `personalize_choices`
+* `sn_si.basic`
+* `x_elas2_sir_int.integration_user`
+. Click *Submit* to create the user.
+
+*Create a CORS rule*
+
+A CORS rule is required for communication between the Elastic and ServiceNow. To create a CORS rule:
+
+. Go to *System Web Services -> CORS Rules*.
+. Click *New*.
+. Configure the rule settings to use the Elastic SIR API and the Kibana URL:
+* *REST API*: Choose `Elastic SIR API [x_elas2_sir_int/elastic_api]`.
+* *Domain*: Enter the Kibana URL.
+. Go to the *HTTP methods* tab and select *GET*. This chooses the GET HTTP method.
+. Click *Submit* to create the rule.
+
+[float]
+[[servicenow-sir-connector-update]]
+==== Update a deprecated ServiceNow SecOps connector
+
+ServiceNow SecOps connectors created in 7.15 version are marked as deprecated after you upgrade to 7.16. Deprecated connectors have a yellow icon after their name and display a warning message when you select them.
```

Comment:
Unless this only applies to upgrades from 7.15 to 7.16, maybe this: "ServiceNow SecOps connectors created with a previous version of Elastic are marked as deprecated after you upgrade to a new version."

Reply:
Good point, this applies to stack upgrades from version <=7.15 to version >=7.16.